Design Patterns/Singleton/Program.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Singleton
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            DatabaseConnection db = DatabaseConnection.OpenConnection();
```

Comment:
Opening connection to database usually requires ConnectionString parameter. Can you please add such parameter to AppSettings.json file, read it from this file and pass here?